tests/by-util/test_tail.rs

```diff
@@ -2291,7 +2754,9 @@ fn test_follow_name_move2() {
         assert_eq!(buf_stdout, expected_stdout);
         assert_eq!(buf_stderr, expected_stderr);
 
-        args.push("--use-polling");
+        if i == 0 {
+            args.push("--use-polling");
+        }
```

Comment:
Was this just a mistake left over from before?

Reply:
Don't know what to say. Here's the output of clippy
```bash
$ cargo clippy --features tail --no-default-features --test tests -- -D warnings
...

error: it looks like the same item is being pushed into this Vec                                                                                                                   
    --> tests/by-util/test_tail.rs:2294:9                                                
     |                                                                                   
2294 |         args.push("--use-polling");  
     |         ^^^^               
     |                                      
     = note: `-D clippy::same-item-push` implied by `-D warnings`                        
     = help: try using vec!["--use-polling";SIZE] or args.resize(NEW_SIZE, "--use-polling")                                                                                        
     = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#same_item_push                                                                
                                                                                         
error: it looks like the same item is being pushed into this Vec                                                                                                                   
    --> tests/by-util/test_tail.rs:2454:9                                                
     |                                                                                                                                                                             
2454 |         args.push("--use-polling");                                                                                                                                         
     |         ^^^^                                                                      
     |                              
     = help: try using vec!["--use-polling";SIZE] or args.resize(NEW_SIZE, "--use-polling")                                                                                        
     = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#same_item_push                                                                
                                            
error: could not compile `coreutils` due to 2 previous errors
```

I haven't found the original suggestions of clippy appropriate.

Reply:
Hmm maybe I forgot `-D warnings`. Still, it's a change in logic right?

Reply:
> Still, it's a change in logic right?

Not here, I guess. The push into the args vec takes place at the end of the iteration, so the second push (which I changed to not happen anymore) is never applied. Even if so, using `--use-polling` twice would produce an usage error. I'm pretty sure this was a mistake when setting up the test case.

Reply:
Can't help it but I guess we've got the same version problem here.

Reply:
Found a CI run where you can see this failing in the ci:style/lint:ubuntu-latest https://github.com/uutils/coreutils/actions/runs/3526285240/jobs/5914032057

Reply:
Maybe the fix wasn't good enough. Let's leave it